automation_tools/__init__.py

```diff
@@ -3245,3 +3245,33 @@ def setup_rhv_ca():
         '/etc/pki/ca-trust/source/anchors/'.format(http_server))
     run('update-ca-trust enable ; update-ca-trust')
     print("RHV CA cert has been successfully added to CA trust")
+
+
+def configure_telemetry():
+    """Setup telemetry on the satellite box with grafana monitoring tool
+    """
+    http_server = os.environ.get('HTTP_SERVER_HOSTNAME')
+    print("Install required packages for PCP")
+    run('yum -y install pcp pcp-pmda-apache')
+    run('wget {0}/pub/configure_telemetry.sh'.format(http_server))
+    print("Configure Apache and hotproc")
+    run('chmod 777 configure_telemetry.sh && ./configure_telemetry.sh')
+    run('cd /var/lib/pcp/pmdas/proc/ &&  c | ./Install')
+    run('cd /var/lib/pcp/pmdas/apache && 80 | ./Install')
```

Comment:
ditto

Reply:
```
# echo 90 | ./Install 
Apache port number [80]? Updating the Performance Metrics Name Space (PMNS) ...
Terminate PMDA if already installed ...
```
passing value this way doesn't work and we are using default anyway so just echo (causing newline) is enough
```echo | ./Install``` or ```./Install <<< ''```